Cython/Utility/ObjectHandling.c

```diff
@@ -1992,6 +1992,26 @@ typedef struct {
 /////////////// UnpackUnboundCMethod ///////////////
 //@requires: PyObjectGetAttrStr
 
+#define __PYX_REINTERPRET_FUNCION(func_pointer, other_pointer) ((func_pointer)(void(*)(void))(other_pointer))
+#define __PYX_REINTERPRET_POINTER(pointer_type, pointer) ((pointer_type)(void *)(pointer))
+#define __PYX_RUNTIME_REINTERPRET(type, var) (*(type *)(&var))
```

Comment:
You seem to be using only the first of these three. We shouldn't commit unused code. It's really nice to know about these, though. We could add them to `ModuleSetupCode.c` instead and comment them out with `//`, so that the unused ones don't take any space in the generated C files, but are still available if they start becoming useful. (There are probably places already where they could be applied.)